core/server/helpers/get.js

```diff
@@ -184,6 +191,22 @@ get = function get(resource, options) {
         logging.error(err);
         data.error = err.message;
         return options.inverse(self, {data: data});
+    }).finally(function () {
+        const end = new Date();
+        const totalMs = end.valueOf() - start.valueOf();
+        const logLevel = config.get('logging:slowHelper:level');
+        const threshold = config.get('logging:slowHelper:threshold');
+        if (totalMs > threshold) {
+            logging[logLevel](new errors.HelperWarning({
```

Comment:
maybe not a big deal, but we might want to protect against undefined logLevel w/ smth like:
```js
const log = (logging[logLevel] || logging.warn).bind(logging);
log(new errors.HelperWarning........
```

Reply:
`undefined` _shouldn't_ happen unless you explicitly set it to undefined because it's set in the default fallback config already. I figured if you're setting an invalid value then it's probably better for that to blow up than do something unexpected. However, seeing as the 💥 may be pretty delayed maybe it's better to be defensive in this case and check for a valid value 🤔

Reply:
> I figured if you're setting an invalid value then it's probably better for that to blow up than do something unexpected.

Agree, and also - this is a pretty low level config option, I don't think people would be randomly tweaking it.

I don't think it's critical, but your call :relaxed:

Reply:
I looked at it but the code gets kinda bloated for something which 99.9% of users are unlikely to touch so I'll leave it as-is. Plus who want's error handling for their error handlers 😄